.github/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Android Build Pipeline
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ main ]
```

Comment:
I think this should also be `master` unless we rename our primary branch.

Reply:
Ah probably you are right. Fixed.